_data/affiliations.yml

```diff
@@ -1,6 +1,5 @@
 - name: The University of Manchester
   image_url: /images/institutions/logo-university-of-manchester.svg
-  pid: https://ror.org/02aswte26
```

Comment:
I think this link to the ROR might be worth keeping in - and is in line with the linking out to registries approach of RSQKit